### DISCUSSION TITLE
Border around plugin manager and terminal window

### DISCUSSION THREAD

**Discussion by CryoViking** [LazyVim/LazyVim#2106] [Q&A] [answered]
I've got a semi-configured, mostly vanilla LazyVim setup that I'm running neovim with at the moment. I was wondering if there is some config option or something I can set to place a border around the floating window that pops up when <C-/> or <C--> is pressed (for the terminal) or <leader>l for the plugin manager?

If there isn't a way to do it within LazyVim, does anyone know of any plugins that I could potentially use to achieve the desired look when opening the  floating terminal window

**Comment by dpetka2001** [comment, marked as answer]
See #2092.

**Reply by CryoViking** [reply]
Thanks, do you know if there is way it can also have a semi-transparent background?

**Reply by dpetka2001** [reply]
Not that I know of. Many colorschemes, like tokyonight which is the Lazyvim default, support transparency. Then if you want to have semi-transparency I would tinker with the opacity option of the terminal you use. For example, I use transparent tokyonight and then set opacity to 0.5 in my kitty terminal. That gives the effect of semi-transparency. Otherwise, you would have to maybe change the colorscheme you use directly, which I find a bit tedious for my liking. It's easier to set terminal opacity to a percentage that gives the effect of semi-transparency.

**Reply by CryoViking** [reply]
I've already got transparency on nvim and opacity set on my terminal, I was hoping to get a more opaque look for floating windows so that they "pop" more when on top, the border helps, was just hoping for the extra little more oomph. That said I can work with what I have at the moment. Thanks for the answers

**Reply by dpetka2001** [reply]
Sorry, can't help you there. There might be another solution that I'm not aware of. Maybe someone more knowledgeable will step forth and suggests you a solution for your needs.

**Reply by dpetka2001** [reply]
Can you try setting `:set winblend=30` for example or some other value? Does this help you achieve what you want? This affects all floating windows however.

**Reply by CryoViking** [reply]
Nah that didn't achieve what I am looking for. I've got other configuration stuff I want to get done in the mean time. I'll come back to this problem when it annoys me sometime in the future.
